docs/source/_templates/about.html

```diff
@@ -0,0 +1,10 @@
+{% include "!about.html"%}
+
+<p>
+<a href="https://circleci.com/gh/Yelp/pip-faster">
+    <img
+        alt="https://circleci.com/gh/Yelp/venv-update.svg?style=shield"
+        src="https://circleci.com/gh/Yelp/venv-update.svg?style=shield"
```

Comment:
No idea. It's in the alabaster theme code. I'm just following existing style.

https://github.com/bitprophet/alabaster/blob/master/alabaster/about.html#L33